spec/namespaces/security.yaml

```
@@ -2255,35 +2255,35 @@ components:
     security.create_role::path.role:
       name: role
       in: path
-      description: The name of the role to be created.
+      description: The name of the role to create.
       schema:
         type: string
       required: true
     security.create_role_mapping::path.role:
       name: role
       in: path
-      description: The name of the role to create a role mapping for.
+      description: The name of the role to create a role mapping.
```

Try my best not to end sentences in prepositions. Going with this instead.

```suggestion
      description: The name of the role for which to create a role mapping.
```